ftw/simplelayout/tests/test_contentpage_behavior.py

```diff
@@ -41,3 +42,13 @@ def test_title_shows_if_show_title_is_true(self, browser):
         self.assertEqual(
             ['A page'],
             browser.css('.documentFirstHeading').text)
+
+    @browsing
+    def test_show_hide_title_field_is_only_for_admins(self, browser):
```

Comment:
Ok. I'm not sure about what the best default is.

As an integrator I'd probably expect that the feature is available for all editing users, not just admins.

From the lawgiver perspective I think the action group `manage content settings` would best match.

How about:
1. Move the permission to `manage content settings`
2. No matter what's in the `rolemap.xml`, override the permission with `self.portal.manage_permission` in the test so that we have an isolated test scenario which does not take presumptions about how the feature should be used.